eth_defi/gmx/types.py

```diff
@@ -0,0 +1,43 @@
+"""
+GMX protocol type definitions.
+
+Common type aliases used throughout the GMX module for better type safety and documentation.
+"""
+
+from typing import TypeAlias
+
+#: Token symbol (e.g., "BTC", "ETH", "USDC")
+TokenSymbol: TypeAlias = str
+
+#: Market symbol (e.g., "BTC", "ETH/USD", "DOGE")
+MarketSymbol: TypeAlias = str
+
+#: Market address as hexadecimal string
+MarketAddress: TypeAlias = str
+
+#: USD-denominated price or value
+USDAmount: TypeAlias = float
+
+#: Annual Percentage Rate as decimal (e.g., 0.05 for 5%)
+APRDecimal: TypeAlias = float
+
+#: Position side identifier ("long" or "short")
+PositionSide: TypeAlias = str
```

Comment:
Here, instead of `str` you can use `Literal["long", "short"]` and edit time syntax highlighting will complain if you try to enter wrong string.